03-client-api/03-nodejs.md

```diff
@@ -23,9 +23,9 @@ npm install grakn-client
 ```
 
 ## Quickstart
-First make sure, the [Grakn server](/docs/running-grakn/install-and-run#start-the-grakn-server) is running..
+First make sure that the [Grakn server](/docs/running-grakn/install-and-run#start-the-grakn-server) is running.
 
-In your source, require `grakn-client`.
+In your source, make `grakn-client` a requirement.
```

Comment:
I would suggest this change:

```suggestion
In your source, add `grakn-client`.
```


I think they read well as a whole:

---

In your source, add `grakn-client`.

```
const GraknClient = require("grakn-client");
```
---

Reply:
After a discussion with Soroush, it appears that this is something that is commonly said in programming context. In Python or Java, the keyword is `import` and people would often say "In your source `import` `grakn-client`".

I would propose we keep this for now and get the PR merged, and then we can have another discussion in order to tweak it even better.

Reply:
```suggestion
In your source, require `grakn-client`.
```

Reply:
Thanks guys!